lib/opencensus/plug/trace.ex

```diff
@@ -83,7 +83,16 @@ defmodule Opencensus.Plug.Trace do
 
         :ocp.with_child_span(span_name(conn), attributes)
 
-        Plug.Conn.register_before_send(conn, fn conn ->
+        encoded =
+          :ocp.current_span_ctx()
+          |> :oc_span_ctx_header.encode()
+          |> :erlang.iolist_to_binary()
+
+        Logger.metadata(tracespan: encoded)
+
+        conn
+        |> Plug.Conn.put_resp_header(header, encoded)
```

Comment:
`header` is unbound here, and also is expected to be lowercase in Plug.
```suggestion
        |> Plug.Conn.put_resp_header(String.downcase(header), encoded)
```